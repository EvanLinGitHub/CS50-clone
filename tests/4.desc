get_string - prints formatted prompt
