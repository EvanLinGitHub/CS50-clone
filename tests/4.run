./get_string_formatted_prompt < 4.in
