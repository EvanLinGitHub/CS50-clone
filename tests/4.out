Prompt #1: foo
Prompt #2: bar
